src/applications/disability-benefits/all-claims/tests/disability-benefits-helpers.js

```diff
@@ -387,92 +141,63 @@ function initPaymentInformationMock(token) {
   });
 }
 
-// Possibly used outside of flow to, and including, 4142
-// function completeApplicantInformation(client, data) {
-//   client
-//     .fillName('root_fullName', data.fullName)
-//     .selectDropdown('root_gender', data.gender)
-//     .fill('input[name="root_socialSecurityNumber"]', data.socialSecurityNumber)
-//     .fillDate('root_dateOfBirth', data.dateOfBirth)
-//     .selectDropdown('root_serviceBranch', data.serviceBranch);
-// }
-
-// function completeVeteranAddressInformation(client, data) {
-//   const { addressLine1, city, state, zipCode } = data.veteran.mailingAddress;
-
-//   client
-//     .fill(
-//       'input[name="root_veteran_mailingAddress_addressLine1"]',
-//       addressLine1,
-//     )
-//     .fill('input[name="root_veteran_mailingAddress_city"]', city)
-//     .selectDropdown('root_veteran_mailingAddress_state', state)
-//     .fill('input[name="root_veteran_mailingAddress_zipCode"]', zipCode);
-// }
-
-// function completeHomelessness(client, data) {
-//   client.selectYesNo(
-//     'root_veteran_homelessness_isHomeless',
-//     data.veteran.homelessness.isHomeless,
-//   );
-// }
+const defaultData = {
```

Comment:
Eh, these "new" chunks aren't actually new. What I did was I just removed all the bits that I saw the puppeteer test was using, deleted the rest of the file, and moved them back. These just happened to be out of order. :man_shrugging: